### ISSUE TITLE
Include files not being copied

### ISSUE THREAD

**Issue by pohzhiee** [ros2/robot_state_publisher#30]
The include directory is not being copied to the target, which causes the lack of a `robot_state_publisher` folder in `/opt/ros/dashing/include`.
This is due to the lack of the following in the CMakeLists.txt
```
install(DIRECTORY include/${PROJECT_NAME}/
  DESTINATION include/${PROJECT_NAME}
)
```
Is this intentional and if so is there some reason behind this? I noticed this only happens in ROS2

**Comment by jacobperron** [comment]
It's possible it was just lost in translation. Here's the commit where it was removed: https://github.com/ros2/robot_state_publisher/commit/4988ec4dbf413c6ec2518991d96254f4869ed954#diff-af3b638bc2a3e6c650974192a53c7291).

I'm okay with adding it back. ~~We could also use the more modern [target_include_directories](https://github.com/ros2/robot_state_publisher/commit/4988ec4dbf413c6ec2518991d96254f4869ed954#diff-af3b638bc2a3e6c650974192a53c7291) instead.~~

@pohzhiee Would you like to open a PR for this?

**Comment by pohzhiee** [comment]
I am not very familiar with `target_include_directories` but from what I understand it only adds the include directories to be searched during compilation. I am not sure how that would solve the issue here which is the include directory not being generated in the output.
Anyway I am using rclcpp's CMakeLists.txt as reference for doing this.

**Comment by jacobperron** [comment]
> I am not very familiar with target_include_directories but from what I understand it only adds the include directories to be searched during compilation. 

You're correct. I was mixed up, ignore my comment :)

**Comment by jacobperron** [comment]
Closed in #31
